packages/react/src/internal.tsx

```diff
@@ -1,4 +1,5 @@
-export * from './hooks/useAuth';
+export { useAuth } from '@aws-amplify/ui-react-core';
```

Comment:
This is where I re-export `useAuth` from `@aws-amplify/ui-react`.

Reply:
```suggestion
// re-export `useAuth` for Studio usage
export { useAuth } from '@aws-amplify/ui-react-core';
```